plugins/out_stackdriver/stackdriver.c

```diff
@@ -443,13 +444,29 @@ static int get_severity_level(severity_t * s, const msgpack_object * o,
     return -1;
 }
 
+static void pack_json_payload(bool operation_extracted, msgpack_packer* mp_pck, 
+                                msgpack_object *obj)
+{
+    /* Specified fields include operation, sourceLocation ... */
+    /* TODO: deal with other fields later */
+
+    if (operation_extracted == true) {
+        pack_object_except_operation(mp_pck, obj);
+    }
+    else {
+        msgpack_pack_object(mp_pck, *obj);
+    }
+}
+
 static int stackdriver_format(const void *data, size_t bytes,
                               const char *tag, size_t tag_len,
                               char **out_data, size_t *out_size,
                               struct flb_stackdriver *ctx)
 {
     int len;
     int array_size = 0;
+    /* The defaulf value is 3: timestamp, jsonPayload, logName. When the operation exists, entry_size will increase by 1 */
```

Comment:
```suggestion
    /* The default value is 3: timestamp, jsonPayload, logName. */
```

You do not need to comment on operation. The reason is that, you need to maintain the comments every time you add new fields, and it become easy to get out-of-date.